test/Microsoft.ML.TestFramework/Attributes/AttributeHelpers.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.TestFramework.Attributes
+{
+    internal static class AttributeHelpers
+    {
+        internal static bool CheckLibcVersionGreaterThanMinimum(double minVersion)
+        {
+#if !NETFRAMEWORK
```

Comment:
Why the `#if` here? The `if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))` will catch anytime we are running on Windows, and return `false`.

Reply:
The reason behind this is that ```Marshal.PtrToStringUTF8``` is not available for netframework which leads to compile error